managed-ledger/src/main/java/org/apache/bookkeeper/mledger/util/RangeCache.java

```diff
@@ -116,7 +148,11 @@ V getValue(K key) {
                 localValue = this.value;
                 lock.unlockRead(stamp);
```

Comment:
don't we need to release this in try-finally block?

Reply:
In this case, that's not necessary. In this class, when locks are held, only field assignments are performed.